### PR TITLE
Sonic Buffs Traitor Borgs

### PR DESCRIPTION
**Pull request by SonicGotNuked**
Readds the ancient Reset Identity Codes command.
Allows traitor borgs to toggle their overclock

### CODE REVIEW

**Comment by SonicGotNuked** [comment]
!review

File: code/game/antagonist/station/traitor.dm [hunk 1]
```diff
@@ -87,7 +87,12 @@ var/datum/antagonist/traitor/traitors
 		add_law_zero(traitor_mob)
 		if(isrobot(traitor_mob))
 			var/mob/living/silicon/robot/R = traitor_mob
+			R.overclockavailable = 1
 			R.emagged = 1
+			..()
```
Comment (Alberyk) [inline code/game/antagonist/station/traitor.dm]: What is this for?

File: code/game/machinery/computer/robot.dm [hunk 2]
```diff
@@ -61,8 +61,11 @@
 		// Antagonistic cyborgs? Left here for downstream
 		if(target.mind && target.mind.special_role && target.emagged)
 			to_chat(target, "Extreme danger.  Termination codes detected.  Scrambling security codes and automatic AI unlink triggered.")
+			target.say("WARNING: Self-Destruct initiated. Unit BZT- [target] will. TERMINATING COMMAND.. NIN- NINE- NINE SECONDS")
 			target.ResetSecurityCodes()
-
+			sleep(20)
```
Comment (Alberyk) [inline code/game/machinery/computer/robot.dm]: No, use timers instead of sleep.

File: code/game/antagonist/station/traitor.dm (hunk 1 above)
Comment (SonicGotNuked) [reply code/game/antagonist/station/traitor.dm]: Removed

**Pull request merged**